src/SwiftReflector/Extensions.cs

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin;
+using System.Text;
+using SyntaxDynamo;
+using System.Collections;
+
+namespace SwiftReflector {
+	public static class Extensions {
+		public static IEnumerable<T> Yield<T> (this T elem)
+		{
+			yield return elem;
+		}
+
+		public static bool IsSwiftEntryPoint (this NListEntry entry)
+		{
+			return !String.IsNullOrEmpty (entry.str) && (entry.str.StartsWith ("__T", StringComparison.Ordinal) ||
```

Comment:
`__T` and `_$s` are legacy

Reply:
According to the ABI document, the `_$s` is used for globals: https://github.com/apple/swift/blob/main/docs/ABI/Mangling.rst#L11